sparsity/dask/core.py

```diff
@@ -59,6 +59,36 @@ def finalize(results):
     return sp.SparseFrame.vstack(results)
 
 
+def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed"):
+    for df in dfs:
+        if not isinstance(df, Delayed):
+            raise TypeError("Expected Delayed object, got %s" % type(df).__name__)
+
+    if meta is None:
+        meta = delayed(make_meta)(dfs[0]).compute()
```

Comment:
I'd raise a warning here, this computation might be very expensive...

Reply:
It's exactly like this in dask.